packages/paste-website/src/pages/tokens/design-tokens-package.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Design Tokens Package
+package: '@twilio-paste/design-tokens'
+description: The Paste design tokens package allows you to install design tokens in a variety of different formats
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/design-tokens/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteDesignTokensPackage(filter: {name: {eq: "@twilio-paste/design-tokens"}}) {
+      edges {
+        node {
+          name
+          description
+          version
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Design Tokens package"
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/button"
+  data={props.data.allPasteDesignTokensPackage.edges}
+/>
+
+***
+
+<content>
+
+The Paste design tokens are available as an NPM package to download and install the latest design tokens and values from a central source of truth. A simple way to grab or view the latest files is to visit [https://unpkg.com/browse/@twilio-paste/design-tokens/dist/](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/). Tokens are supported for different formats within the NPM package:
+
+- [JSON](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.json)
+- [Less](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.less)
+- Scss
+  - [as individual variables](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.default.scss)
+  - [as an scss map](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.map.scss)
+- [CSS custom properties](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.custom-properties.css)
+- [Android xml](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.android.xml)
+- [iOS JSON](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.ios.json)
+- JavaScript
+  - [commonJS](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.common.js)
+  - [es2015](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.es6.js)
+- Typescript definitions
+  - [commonJS](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.common.d.ts)
+  - [es2015](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/tokens.es6.d.ts)
+- [Sketch palette](https://unpkg.com/browse/@twilio-paste/design-tokens/dist/sketch.sketchpalette)
+
+## Themes
+
+Paste supports multiple themes. This is done by exporting the same token formats for each theme. Each file in the root of the design tokens package is exported for each theme with the corresponding values updated for each token. For example:
+
+| Default theme                   | SendGrid theme                  |
+|---------------------------------|---------------------------------|
+| $color-brand-highlight: #F22F46 | $color-brand-highlight: #3368FA |
```

Comment:
Would be cool to put a second example here for a non-color token, like spacing and/or font for the full effect